src/theme/Footer/index.js

```diff
@@ -82,8 +82,16 @@ function Footer() {
         dangerouslySetInnerHTML={{
           __html: copyright
         }} /> : null}
+        <div className="footer__legal" >
+          <ul>
+            <li><a href="https://unity.com/legal">Legal</a></li>
+            <li><a href="https://unity.com/legal/privacy-policy">Privacy Policy </a></li>
+            <li><a href="https://unity.com/legal/cookie-policy">Cookies</a></li>
+            <li><a href="https://unity.com/legal/do-not-sell-my-personal-information">Do Not Sell or Share My Personal Information</a></li>
+         </ul>
+        </div>
         <div className="footer__details" >"Unity", Unity logos, and other Unity trademarks are trademarks or registered trademarks of Unity Technologies or its affiliates in the U.S. and elsewhere (<a href="https://unity3d.com/legal/trademarks">more info here</a>). Other names or brands are trademarks of their respective owners.</div>
```

Comment:
The `unity3d.com` content has migrated over to `unity.com`.

Please use the new URL for the trademark content. See https://unity.com footer for reference.